addons/project_mrp/data/project_mrp_demo.xml

```diff
@@ -0,0 +1,7 @@
+<odoo>
+    <data noupdate="1">
+        <record id="mrp.mrp_production_3" model="mrp.production">
+            <field name="analytic_account_id" ref="project.analytic_office_design"/>
+        </record>
+    </data>
+</odoo>
```

Comment:
There is a missing blank line :)

Reply:
Fixed, thank you :slightly_smiling_face: